client/src/components/Footer/FooterStyle.css

```diff
@@ -0,0 +1,45 @@
+html{
+    height:100%;
```

Comment:
Is this needed? This will affect all other pages

Reply:
I will remove it